amundsen_application/static/js/components/ColumnList/ColumnStats/index.tsx

```diff
@@ -24,12 +24,10 @@ const ColumnStatRow: React.FC<ColumnStatRowProps> = ({
   stat_type,
   stat_val,
 }: ColumnStatRowProps) => {
-  const numberVal = +stat_val;
-
   return (
     <div className="column-stat-row">
       <div className="stat-name body-3">{stat_type.toUpperCase()}</div>
-      <div className="stat-value">{formatNumber(numberVal)}</div>
+      <div className="stat-value">{Number(stat_val) ? formatNumber(+stat_val) : stat_val}</div>
```

Comment:
The approaches mentioned here are more ideal/standard ways of checking this:
https://stackoverflow.com/questions/175739/built-in-way-in-javascript-to-check-if-a-string-is-a-valid-number

Reply:
I am getting a type error when trying to use isNaN with Ts because it is expecting a number type

Reply:
Oh, here's another reference: https://stackoverflow.com/questions/42120046/typescript-isnan-only-accepts-a-number
```
if (typeof expectedValue === "string" && !Number.isNaN(Number(expectedValue))) {
    expectedValue = Number(expectedValue);
}
```